userspace/libsinsp/filter.cpp

```diff
@@ -1209,6 +1240,50 @@ bool sinsp_filter_check::flt_compare(cmpop op, ppm_param_type type, void* operan
 				}
 			}
 			return false;
+		case PT_CHARBUFARRAY:
+			strvect = (vector<char*>*)*(uint64_t *)operand1;
+			if (op == CO_IN)
+			{
+				for (auto it = strvect->begin(); it != strvect->end(); ++it)
+				{
+					size_t len = strlen(*it);
+					filter_value_t item((uint8_t *) *it, len);
+					if(len < m_val_storages_min_size ||
+						len > m_val_storages_max_size ||
+						m_val_storages_members.find(item) == m_val_storages_members.end())
+					{
+						return false;
+					}
+				}
+				return true;
+			}
+			else if (op == CO_INTERSECTS)
+			{
+				for (auto it = strvect->begin(); it != strvect->end(); ++it)
+				{
+					size_t len = strlen(*it);
+					filter_value_t item((uint8_t *) *it, len);
+					if(len >= m_val_storages_min_size &&
+						len <= m_val_storages_max_size &&
+						m_val_storages_members.find(item) != m_val_storages_members.end())
+					{
+						return true;
+					}
+				}
+				return false;
+			}
+			else if (op == CO_PMATCH)
```

Comment:
I know that pmatch was supported for json events (falco json_evt.cpp), but I don't see it used in the k8s audit rules, and it's kind of complicated to explain-- "(/some/path, /other/path) pmatch (/bin, /usr) returns true if *any* value on the left hand side has any of the values on the right hand side as a prefix". Do you think this is too complex to bother supporting?